fixtures/cruft/_git/COMMIT_EDITMSG

```diff
@@ -0,0 +1 @@
+Test commit
```

Comment:
I think we can remove all unnecessary files for this fixture like `logs/*`, `object/*` (except packs), `refs/tags/*` etc

Reply:
btw, if I remove `tags`/`remotes` directories out the tool explodes. I put `.gitkeep` there for now, to preserve the directories on checkout.

Reply:
okay, `.gitkeep` is also not a great idea here, gonna look up a better compromise

Reply:
added checks whether dir exists before reading for refs/tags in https://github.com/discoveryjs/scan-git/pull/5/commits/4265b6e3529671dff79c35972e5842c60f05b38d